.gitignore

```diff
@@ -7,4 +7,5 @@ __pycache__
 build/
 dist/
 skywalking_python.egg-info/
-*.pyc
\ No newline at end of file
+*.pyc
+/apache_skywalking.egg-info/
```

Comment:
Please remove the `skywalking_python.egg-info` (at line 9), I've forgot to rename it after I rename the module name, thanks

Reply:
ok